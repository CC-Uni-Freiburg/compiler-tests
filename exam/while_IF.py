# Author: Pascal Walter 4924063
x = input_int()
if x > 42:
    y = 0
    while y < 42:
        y = y + 1
else:
    while x < 42:
        x = x + 1

print(x)

x = 10
while x >= -10:
    if x > 0:
        print(x)
    else:
        print(x * (-1))
    x = x - 1
