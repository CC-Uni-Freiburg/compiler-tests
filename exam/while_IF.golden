4210987654321012345678910
